github/GithubIntegration.py

```diff
@@ -22,13 +22,15 @@ class GithubIntegration:
     # keep non-deprecated arguments in-sync with Requester
     # v2: remove integration_id, private_key, jwt_expiry, jwt_issued_at and jwt_algorithm
     # v2: move auth to the front of arguments
-    # v2: add * before first argument so all arguments must be named,
+    # v2: move * before first argument so all arguments must be named,
     #     allows to reorder / add new arguments / remove deprecated arguments without breaking user code
+    #     added here to force named parameters because new parameters have been added
     def __init__(
         self,
         integration_id=None,
         private_key=None,
         base_url=Consts.DEFAULT_BASE_URL,
+        *,
         timeout=Consts.DEFAULT_TIMEOUT,
```

Comment:
Without this, existing code like `GithubIntegration(id, key, url, expiry)` would silently use `expiry` as `timeout`. This breaks code that uses `jwt_*` arguments as positional arguments.

Reply:
I actually don't know what this syntax means. Could you point to a doc that explains it?